index.js

```diff
@@ -1008,31 +1053,42 @@ Multi.prototype.send_command = function (command, args, index, cb) {
     });
 };
 
-Multi.prototype.exec = Multi.prototype.EXEC = function (callback) {
+Multi.prototype.exec_transaction = function (callback) {
     var self = this;
+    var len = this.queue.length;
+    var cb;
```

Comment:
I added Multi.exec_atomic (I can't get used to snack case^^). It'll use the batch.exec function if none or one command is used and therefor return without fireing any command to redis.